doc/manual/variables-and-scoping.rst

```diff
@@ -285,69 +252,88 @@ writing:
 
 An explicit ``global`` is needed to assign to a global variable:
 
-.. doctest::
+.. sidebar:: Avoiding globals
 
-    julia> x = 1;
+    Avoiding changing the value of global variables is considered by many
+    to be a programming best-practice.
+    One reason for this is that remotely changing the state of global variables in other
+    modules should be done with care as it makes the local behavior of the program hard to reason about.
+    This is why the scope blocks that introduce local scope require the ``global``
+    keyword to declare the intent to modify a global variable.
 
-    julia> function foobar()
-               global x = 2
-           end;
+.. doctest::
```

Comment:
This doctest appears to belong following the statement "An explicit ``global``..." on line 253. But due to the deletions above neither `foobar` nor `x` associated with the remaining parts of this doctest are defined?